.github/workflows/deploy_and_build_prod.yml

```
@@ -109,9 +110,15 @@ jobs:
           RELEASE_TAG=$(echo $(git describe --tags --abbrev=0))
           # Using param substitution, we output x.x.x instead
           echo "::set-output name=version::${RELEASE_TAG#*-}"
+      - name: Get Latest Build Version 
+        id: latestBuildVersion
+        run: |
+          # there were 116 builds before the run count got resetted due to migrating to eas
+          sum=$(( $github.run_number + 116 ))
```

hmm, i suspect the syntax is this or the other
```suggestion
          sum=$(( $GITHUB_RUN_NUMBER + 116 ))
```

```suggestion
          sum=$(( ${{ github.run_number }} + 116 ))
```

---

not very sure of the (( ))